File: clients/csharp/tests/NKTgClientTests.cs
using Xunit;
using NKTgLaw;

namespace NKTgLawTests
{
    public class NKTgClientTests
    {
        [Fact]
        public void TestNKTg1()
        {
            var client = new NKTgClient(2, 3, 5, 0.1);
            Assert.Equal(15.2, client.NKTg1(), 1);
        }

        [Fact]
        public void TestNKTg2()
        {
            var client = new NKTgClient(2, 3, 5, 0.1);
            Assert.Equal(3.04, client.NKTg2(), 2);
        }
    }
}

File: clients/swift/Tests/NKTgClientTests.swift
import XCTest
@testable import NKTgLaw

final class NKTgClientTests: XCTestCase {
    func testNKTg1() {
        let client = NKTgClient(x: 2.0, v: 3.0, m: 5.0, dmDt: 0.1)
        XCTAssertEqual(client.nktg1(), 15.2, accuracy: 1e-6)
    }

    func testNKTg2() {
        let client = NKTgClient(x: 2.0, v: 3.0, m: 5.0, dmDt: 0.1)
        XCTAssertEqual(client.nktg2(), 3.04, accuracy: 1e-6)
    }
}
